src/openapi/spec/unleash-context-schema.ts

```diff
@@ -10,7 +10,9 @@ export const schema = {
         remoteAddress: { type: 'string' },
```

Comment:
Ideally this schema `schema` should have additional properties defined as something like this:

```ts
    additionalProperties: {
        oneOf: [
            { type: 'string' },
            { type: 'string', format: 'date-time' },
            { type: 'number' },
            {
                type: 'object',
                additionalProperties: {
                    anyOf: [{ type: 'string' }, { type: 'number' }],
                },
            },
            ,
        ],
    },
```

But I can't for the life of me make OpenAPI recognize that as a valid type. I've spent a few hours on just that, but I don't think it's a blocker for now. If you've got any suggestions, I'm more than happy to listen, but otherwise I suggest we leave it as is.

Reply:
I sadly do not have suggestions here but one of the team that's been more involved might have.

I don't think this is a problematic though

Reply:
Agreed. I'll leave it as is for now. We can revisit it later if we need to.